hybrid/financial/custom_financial_model.py

```diff
@@ -178,17 +180,28 @@ def __init__(self,
         self.assign(fin_config)
```

Comment:
No, this assigns any variables that are not included in the six data classes (e.g., batt_annual_discharge_energy) . I remember having a reason for not creating their respective data classes, but I can't recall. It may have just been that the classes would have had only one or two members, I'm not sure.

Reply:
Gotcha thanks!